docs/authentication/session-management.mdx

```diff
@@ -0,0 +1,896 @@
+--data-urlencode "principal_type=application" \
+--data-urlencode "principal_id=$(APPLICATION_ID)" \
+-H "Authorization Bearer $(MANAGEMENT_API_TOKEN)"'
+title="/tokens"
+/>
+
+#### Example Response
+
+```json
+{
+    "tokens": [
+        {
+            "id": "uZcs8hF4_vR69eonor3U_lottYxtSfrX",
+            "scopes": ["tenants:create"],
+            "expires": 1675177245,
+            "issued_at": 1675090845,
+            "token_type": "access",
+            "token_format": "self_contained"
+            "token_suffix": "Jv_1828r9",
+        }
+    ],
+    "total_size": 1
+}
+```
+
+#### Example: List access tokens for an identity
+
+<MultiLanguageCodeBlock
+curl='curl -G "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/applications/$(APPLICATION_ID)/tokens" \
+--data-urlencode "principal_type=identity" \
+--data-urlencode "principal_id=$(IDENTITY_ID)" \
+-H "Authorization Bearer $(MANAGEMENT_API_TOKEN)"'
+title="/tokens"
+/>
+
+#### Example Response
+
+```json
+{
+    "tokens": [
+        {
+            "id": "cKigHwlWRW5h3Dv4CCMBICiqf-j1i1yJ",
+            "scopes": ["myapp:read"],
+            "expires": 1688488934,
+            "issued_at": 1688402534,
+            "token_type": "access",
+            "token_format": "self_contained"
+        },
+        {
+            "id": "Kh-yAOSh9ex7cVuVbVUr84EMmou6VBA-",
+            "scopes": ["myapp:read"],
+            "expires": 1688488934,
+            "issued_at": 1688402534,
+            "token_type": "access",
+            "token_format": "self_contained"
+        }
+    ],
+    "total_size": 2,
+    "next_page_token": null
+}
+```
+
+### JWT Strategy with Refresh/Access Tokens
+
+JSON Web Tokens (JWTs) are a popular choice for authentication and authorization. This strategy involves using two types of tokens: access tokens and refresh tokens.
+
+#### Access Tokens:
+
+- Access tokens are short-lived JWTs issued upon successful authentication.
+
+- They contain information about the user and their permissions (claims).
+
+- The server validates the access token for each request to authorize the user's actions.
+
+#### Refresh Tokens:
+
+- Refresh tokens are long-lived JWTs, used to obtain new access tokens after the previous ones expire.
+
+- Clients can securely request a new access token using the refresh token, avoiding frequent logins.
+
+- Refresh tokens should be stored securely and should not be accessible by client-side scripts.
+
+#### Why/why not to use local storage
+
+- It's NOT recommended to store JWTs in local storage due to security risks.
+
+- Local storage is vulnerable to XSS attacks, allowing malicious scripts to steal tokens.
+
+- Tokens in local storage have a longer lifespan, increasing the window of opportunity for attacks.
+
+- Instead, use HTTP-only cookies with the Secure flag for better security and protection against XSS attacks.
+
+### Session Cookie Strategy
```

Comment:
Similarly to the previous comment, move this up and introduce that there are decisions to make with session artifacts: where and how to store them and how long, and also how to introduce cookies into the mix as BI doesn't have SDK support for this

Reply:
Moved 👍